src/Client/ClientInterface.php

```diff
@@ -39,4 +39,15 @@ interface ClientInterface {
    */
   public function list(array $options = []);
 
+  /**
+   * Gets the resource's main entity by its primary ID.
+   *
+   * @param int|string $id
+   *   The ID of the entity to get.
+   *
+   * @return object
+   *   The parsed response.
+   */
+  public function get($id): object;
+
```

Comment:
**CHANGE REQUEST**

We haven't decided to use PHP 7.2+ syntax for method responses in this module yet, so let's remove that. Also, the expected response here would be the entity object i.e. " The parsed response" => "The remote entity object".